server/channels/api4/group.go

```diff
@@ -301,6 +302,14 @@ func patchGroup(c *Context, w http.ResponseWriter, r *http.Request) {
 	auditRec.AddEventResultState(group)
 	auditRec.AddEventObjectType("group")
 
+	c.App.Srv().GetTelemetryService().SendTelemetryForFeature(
+		telemetry.TrackGroupsFeature,
+		"modify_group__edit_details",
```

Comment:
nit: Should event names also be constants?

Reply:
if we move SKUs from feature to event, then there is a case for it. If there is no other use than sending the event (no comparison and little to no reuse) there is no point in making a constant of something only used once.

If we want to change SKU calculation I can create a separate PR or change it here (0/5)